engine/src/flutter/shell/platform/darwin/ios/BUILD.gn

```diff
@@ -209,97 +209,99 @@ source_set("flutter_framework_source") {
   ]
 }
 
-platform_frameworks_path =
```

Comment:
github is confused with this diff - all these lines are just indented because of the `if` right?